### PR TITLE
Jkeenan/gha 258 win32 20200901

### PR DESCRIPTION
**Pull request by jkeenan**
This p.r. is untested; I have no Windows.  It *may* clean up some `use strict 'vars';` failures.

For: https://github.com/atoomic/perl/issues/258

### CODE REVIEW

File: cpan/Win32API-File/t/file.t [hunk 1]
```diff
@@ -24,29 +25,29 @@ END {print "not ok 1\n" unless $loaded;}
 # the ../lib directory in @INC will no longer work once
 # we chdir() into the TEMP directory.
 
-use Win32;
+#use Win32;
```
Comment (toddr) [inline cpan/Win32API-File/t/file.t]: I don't follow how this can function with these off.

File: cpan/Win32API-File/t/file.t [hunk 2]
```diff
@@ -24,29 +25,29 @@ END {print "not ok 1\n" unless $loaded;}
...
 # we chdir() into the TEMP directory.
 
-use Win32;
+#use Win32;
 use File::Spec;
 use Carp;
 use Carp::Heavy;
 
-use Win32API::File qw(:ALL);
+#use Win32API::File qw(:ALL);
```
Comment (toddr) [inline cpan/Win32API-File/t/file.t]: Accidentally commented?

**Review by toddr — CHANGES_REQUESTED** [review]
Given a branch, I can switch to windows and try this out.

File: cpan/Win32API-File/t/file.t (hunk 2 above)
Comment (jkeenan) [reply cpan/Win32API-File/t/file.t]: Corrected.

**Review by toddr — APPROVED** [review]

**Comment by toddr** [comment]
I think we're good to go.

**Comment by jkeenan** [comment]
Rebased on target branch, merged into target branch, which was just now pushed upstream as commit da0a32ff74ba2ef2686dc14c210eb832fbff472c.

**Pull request closed without merging**